src/fhir/1_Questionnaire-C-IDAS.json

```diff
@@ -106,7 +106,7 @@
         "extension": [
           {
             "url": "http://hl7.org/fhir/StructureDefinition/rendering-xhtml",
-            "valueString": "<h4>Sadness</h4><div>Sad expression, sad voice, tearfulness</div>"
+            "valueString": "<h4>Sadness</h4><div>In the last week have you observed Sadness in your care recipient? (sad expression, sad voice, tearfulness)</div>"
```

Comment:
@sierramatice same here, `text` field should be updated with the updated text (without the html).